app/webpacker/controllers/terms_of_service_banner_controller.js

```diff
@@ -0,0 +1,16 @@
+import { Controller } from "stimulus";
+
+export default class extends Controller {
+  static values = { url: String };
+
+  accept() {
+    const token = document.querySelector('meta[name="csrf-token"]').content;
+    // We don't really care if the update fails, if it fails it will result in the banner still
+    // being shown.
+    fetch(this.urlValue, { method: "post", headers: { "X-CSRF-Token": token } });
+  }
+
+  close_banner() {
+    this.element.remove();
+  }
+}
```

Comment:
If we used a reflex instead of a controller action, would that have made the Stimulus controller obsolete? Then we wouldn't need to care about the token and just rendered morphed the banner to be empty. It would also get rid of the additional route.